static/EncryptedRequests.js

```diff
@@ -0,0 +1,15 @@
+/**
+Returns the original data encrypted using the server's public key.
+*/
+async function encryptData(original) {
+    let result = "";
+    await fetch("/api/get-public-key").then(response => response.json()).then(response => {
+        let publicKey = forge.pki.publicKeyFromPem(response.key);
+        let encrypted = publicKey.encrypt(original, "RSA-OAEP", {
```

Comment:
We may be able to encript the response key directly instead of storing to the let publicKey variable, i.e.: 
let encripted = forge.pki.publicKeyFromPem(response.key). encrypt(original, "RSA-OAEP", {...